inscriptions/ISic3326.xml

```diff
@@ -191,12 +211,6 @@
                         <citedRange>164-165 no.1 fig.2</citedRange>
                         <ptr target="http://zotero.org/groups/382445/items/4MCP2PVV"/>
                     </bibl>
-                    <bibl>
-                        <author>Manganaro</author>
-                        <date>1989</date>
-                        <citedRange>178 n.55 fig.59bis</citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/EZMBJ6GN"/>
-                    </bibl>
```

Comment:
why has this reference been deleted? NB it's reference to the footnote on p.178 (note 55), where this text is presented, together with fig.59bis and I've checked and it is correct.

Reply:
Riiight, it's 'n.' and not 'no.' - tought it had been missplaced as i couldn't find anything about this inscription under no. 55! Sorry, was careless, it's back up again.